pytest.ini

```diff
@@ -2,6 +2,6 @@
 addopts = -v -rf --ff --cov-report=term-missing -m "not integration"
 testpaths =
     tests/unit_tests*
-norecursedirs = xarray_utils
 markers =
-    integration: marks tests as integration (select with '-m "integration"')
\ No newline at end of file
+    integration: marks tests as integration (select with '-m "integration"')
+norecursedirs = xarray_utils
```

Comment:
@biosafetylvl5  @evrose54  I'm assuming we want the -m integration and -m not integration bits ?

Reply:
Yep! This looks good to me